src/main/docker/gitlab-gitlabci.prod.yml

```diff
@@ -0,0 +1,12 @@
+version: "3.6"
+services:
+    gitlab:
+        environment:
+            GITLAB_OMNIBUS_CONFIG: |
```

Comment:
Maybe "prod" is a bit misleading. It is the config for the "remote teste server". I think it is fine to leave that here for now - the real prod config has to be handled by the Admin team.

Reply:
Yes, exactly. It is for test servers. Would it be better if I name it `gitlab-gitlabci.remotetestserver.yml`.

Reply:
I have added a template for an `.env` file in which you can also configure whether a certificate should be used. This also works well with http only. So that there are no further misunderstandings regarding a `prod.yml` file.